librz/include/rz_types_base.h

```diff
@@ -5,6 +5,7 @@
 #include <sys/types.h>
 #include <limits.h>
 #include <math.h>
+#include <fenv.h>
```

Comment:
Is this one necessary?

Reply:
yes for the `feclearexcept` function, but i guess that could be moved into `librz/util/float/float.c`